ETT-Backend/Repository/DBConnection.cs

```diff
@@ -0,0 +1,62 @@
+using Npgsql;
+using System;
+using System.Collections.Generic;
+using Microsoft.Extensions.Configuration;
+using ETT_Backend.Configuration;
+
+namespace ETT_Backend.Repository
+{
+  public class DBConnection : IDisposable
+  {
+    public bool IsConnectionOpen
+    {
+      get
+      {
+        return connection.State == System.Data.ConnectionState.Open;
+      }
+      private set { }
+    }
+    private NpgsqlConnection connection;
+
+    public DBConnection()
```

Comment:
So if we injected AppConfiguration at startup, we could just access it like this DBConnection(IAppConfiguration appConfiguration) (we would need an interface). That way the ServiceCollection in AppConfiguration is the same as throughout the app and not its own thing

Reply:
Fixed AppConfiguration class